int32 flag
string breadType
--- 
int32 response
